src/test/java/com/desiato/puresynth/PureSynthApplicationTests.java

```diff
@@ -1,13 +0,0 @@
-package com.desiato.puresynth;
-
-import org.junit.jupiter.api.Test;
-import org.springframework.boot.test.context.SpringBootTest;
-
-@SpringBootTest
-class PureSynthApplicationTests{
-
-	@Test
-	void contextLoads() {
```

Comment:
This one actually should work. It check that application starts.

Reply:
ok, I restored this class